docs/Interoperator-Features.md

```diff
@@ -127,6 +127,22 @@ $ helm install --set cluster.host=xxxx \
      --namespace interoperator --wait interoperator interoperator
 ```
 
+## Add Custom Labels to all the Service Instance Namespace
+
+You can add custom labels to the namespace created for the service instance. This custom labels can also be use to exploit `Pod Security Admission` feature of Kubernetes. Below is the example shows how you can add labels,
+```
+In Values.yaml, you can find broker.services_namespace_labels where you can add labels of your choice -

```

Comment:
May be this line can be outside of the code block.

Reply:
I think it will be good to mention that the custom labels are applied only when the `broker.enable_namespaced_separation` is set to `true`